spec/tla/run-all-stages.sh

```
@@ -0,0 +1,52 @@
+FAIL=0
+
+run_stage() {
+  local name="$1" tla="$2" cfg="$3"
+  printf "  %-50s " "$name"
+  rm -rf /tmp/tlc-run
+  local output
+  output=$(timeout 300 java -Xmx4g -XX:+UseParallelGC \
+    -cp ~/tools/tla/tla2tools.jar tlc2.TLC \
+    "$tla" -config "$cfg" -workers 4 \
+    -metadir /tmp/tlc-run -noGenerateSpecTE 2>&1)
+  local rc=$?
```

Shellcheck (0.11.0)

[warning] rc appears unused. Verify use (or export if used externally).

(SC2034)